tests/console/check_os_release.pm

```diff
@@ -29,6 +29,7 @@ sub run {
         my $product = get_required_var('SLE_PRODUCT');
         $product = 'sles_sap' if (is_sles4sap and is_sle('<15'));
         $product = 'sles' if (is_sles4sap and is_sle('>=15'));
+        $product = 'sles' if (is_sles4sap and check_var('SCC_REGCODE_SLES4SAP', 'invalid_key') and is_sle('=12-SP5'));
```

Comment:
```suggestion
         $product = 'sles' if ( is_sles4sap and is_sle('>=15')
                                             || (is_sles4sap and check_var('SCC_REGCODE_SLES4SAP', 'invalid_key') and is_sle('=12-SP5')));
```

downside of this suggestion is the longer line 

But maybe we can think about converting the overall block like
```
my $product;
if (is_sles4sap) {
  if  (  $product = 'sles' 
        || (check_var('SCC_REGCODE_SLES4SAP', 'invalid_key') and is_sle('=12-SP5')))  {
       $product = 'sles' 
  } else if (is_sle('<15')) {
        $product = 'sles_sap' ;
   } else if (is_sle('>=15')
                  || ) {
       $product = 'sles' 
  } else {
        $product = get_required_var('SLE_PRODUCT');
 }
else {
   $product = get_required_var('SLE_PRODUCT');
}
```

Reply:
I prefer the 4 lines too it much more clear, so keep the original line.

Reply:
I prefer the 4 lines too it much more clear, so keep the original line.